rust/scx_stats/src/server.rs

```diff
@@ -280,6 +280,27 @@ where
         self.visit_meta(self.top.as_ref().unwrap(), &mut |_| Ok(()))
     }
 
+    pub fn describe_nstats_meta<W: Write>(
+        &self,
+        w: &mut W,
+        stats_meta_names: Option<&[&str]>,
+    ) -> Result<()> {
+        let meta_names = match stats_meta_names {
+            Some(v) => v,
+            None => {
+                return Err(anyhow!(
+                    "Expected stats meta names to look up in server data"
```

Comment:
Especially given that this function doesn't accept `None` as input and most existing users of `describe_meta()` don't specify `from`.